### PR TITLE
XW-27 | Fix unit tests for Ember helpers and add some more

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/XW-27

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
LGTM

File: test/specs/front/scripts/main/helpers/duration-helper.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	});
+
+	QUnit.test('> 60 seconds and < 1 hour', function () {
+		equal(durationHelper([181]), '03:01');
+	});
+
+	QUnit.test('> 1 hour', function () {
+		equal(durationHelper([3661]), '01:01:01');
+	});
+});
```
Comment (bognix) [inline test/specs/front/scripts/main/helpers/duration-helper.js]: how about checking the negative value and 0?

**Comment by bognix** [comment]
the new syntax for QUnit looks great! and finally we have some real unit tests! good job!
LGTM - although you may consider adding test cases for incorrect data (negative values)

File: test/specs/front/scripts/main/helpers/duration-helper.js (hunk 1 above)
Comment (rogatty) [inline test/specs/front/scripts/main/helpers/duration-helper.js]: Good call, we had a bug there. Fixed in https://github.com/Wikia/mercury/commit/46f488bc812b21ed8f45fec820b849b609214e4f
